.github/workflows/ci-markdownlint.yml

```diff
@@ -0,0 +1,22 @@
+name: Markdown Lint Check
+
+on:
+  pull_request:
+
+jobs:
+  markdownlint-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      # equivalent cli: markdownlint-cli2  "**/*.md" "#lambda/.aws-sam/**" "#.github/pull_request_template.md" "#.github/ISSUE_TEMPLATE/bug-or-feature-request.md"  --config .markdownlint.json
+      - name: "Markdown Lint Check"
+        uses: DavidAnson/markdownlint-cli2-action@v16
+        with:
+          fix: false
+          globs: |
+            **/*.md
+            !lambda/.aws-sam/**
+            !.github/pull_request_template.md
+            !.github/ISSUE_TEMPLATE/bug-or-feature-request.md
+          continue-on-error: true
```

Comment:
Seeing this [warning](https://github.com/solarwinds/apm-ruby/actions/runs/10395387675):

```
Warning: Unexpected input(s) 'continue-on-error', valid inputs are ['config', 'fix', 'globs', 'separator']
Run DavidAnson/markdownlint-cli2-action@v[1](https://github.com/solarwinds/apm-ruby/actions/runs/10395387675/job/28787280482?pr=142#step:3:1)
```

Reply:
updated.